service/test/lib/integration/authorizeOpsTest.js

```diff
@@ -24,21 +25,38 @@ const testUserData = {
 const updateUserData = {
   organizationId,
   name: 'Salman',
-  teams: [4]
+  teams: null
 }
 
 
 lab.experiment('AuthorizeOps', () => {
+
+  let wonkaPolicies
```

Comment:
just my mind bugging me: can we move it where also `testUserId` is defined? or move those `let` here? :)